k-distribution/src/main/scripts/lib/pyk/__main__.py

```diff
@@ -81,33 +93,12 @@ def main(commandLineArgs, extraMain = None):
             args['output'].write('\nUnparsed:\n')
             args['output'].write(prettyPrintKast(rule, symbolTable))
 
-    elif args['command'] == 'minimize':
-        json_definition = readKastTerm(kompiled_dir + '/compiled.json')
-        symbolTable = buildSymbolTable(json_definition, opinionated = True)
-        json_term = json.loads(args['json-term'].read())['term']
-        if json_term == KConstant('#Top'):
-            args['output'].write(prettyPrintKast(json_term, symbolTable))
-        else:
-            abstractLabels = [] if args['omit_labels'] is None else args['omit_labels'].split(',')
-            minimized_disjuncts = []
-            for d in flattenLabel('#Or', json_term):
-                dMinimized = minimizeTerm(d, abstractLabels = abstractLabels)
-                (dConfig, dConstraint) = splitConfigAndConstraints(dMinimized)
-                if dConstraint != KConstant('#Top'):
-                    minimized_disjuncts.append(KApply('#And', [dConfig, dConstraint]))
-                else:
-                    minimized_disjuncts.append(dConfig)
-            sorted_disjunct = buildAssoc(KConstant('#Bottom'), '#Or', minimized_disjuncts)
-            new_disjunct = propagateUpConstraints(sorted_disjunct)
-            args['output'].write(prettyPrintKast(new_disjunct, symbolTable))
-
     elif extraMain is not None:
         extraMain(args, kompiled_dir)
 
-    args['output'].flush()
-
     if returncode != 0:
         _fatal('Non-zero exit code (' + str(returncode) + '): ' + str(kCommand), code = returncode)
 
 if __name__ == '__main__':
+    sys.setrecursionlimit(15000000)
```

Comment:
How did you get this number?

Reply:
It's a really big number that is smaller than the max limit on my machine. It's worked OK so far.

Reply:
Worth adding a comment then.

Reply:
Added.